addon/utils/bugsnag-configuration.js

```diff
@@ -14,21 +14,27 @@ export default class BugsnagConfiguration {
         bugsnagInstance[key] = this.config[key];
       });
     } else {
+      /* eslint-disable no-console */
```

Comment:
This is good. I'd be happy to suppress no-console warnings for the entire file if the line comments offend you.